merlin_models/tf/prediction/losses.py

```diff
@@ -0,0 +1,349 @@
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from typing import Tuple
+
+import tensorflow as tf
+from tensorflow.keras.losses import Loss
+
+
+class PairwiseLoss(Loss):
```

Comment:
Maybe we should have a `loss` package, and then have the pair_wise.py that has all the pair-wise losses?

Reply:
Done